src/users/users.service.ts

```diff
@@ -66,11 +65,12 @@ export class UsersService {
 
     if (user) {
       return user;
+    } else {
```

Comment:
```suggestion
    } else {
  
```
ten warunek else jest niepotrzebny bo jeżeli jest user to zwróci usera a jeżeli nie to przejdzie do błędy (tak mi sie wydaje, jeżeli konwencja mówi że ma być else to niech zostanie)